### ISSUE TITLE
SONY Tablet Z2: No Cameras Open

### ISSUE THREAD

**Issue by commonsguy** [commonsguy/cwac-cam2#51]
FFC or RFC, portrait or landscape. Just a progress bar and nothing else.

**Comment by commonsguy** [comment]
LogCat shows:

```
E/CameraDevice-0-LE(10455): Surface with size (w=1920, h=1440) and format 0x1 is not valid, size not in valid set: [1600x1200, 1280x720, 960x720, 720x480, 640x480, 480x320, 320x240, 176x144]
W/CameraDevice-JV-0(10455): Stream configuration failed
E/CameraCaptureSession(10455): Session 0: Failed to create capture session; configuration failed
```

However, the Camera2Basic sample app also has a preview that is not one of the official sizes (comes out as 1920x1128), yet it does not exhibit this problem.

**Comment by commonsguy** [comment]
Fixed, though now the aspect ratios are off, see issue #64.

**Comment by commonsguy** [comment]
Released in v0.2.0
